app/app.js

```diff
@@ -102,13 +102,13 @@ async function createApp(app) {
   // CSS precompiler. needs to come before /static call
   var compileSass = require('express-compile-sass');
   app.use('/css',compileSass({
-      root: `${__dirname}/scss`,
+      root: `${__dirname}/static/scss`,
       sourceMap: true, // Includes Base64 encoded source maps in output css
       sourceComments: true, // Includes source comments in output css
       watchFiles: true, // Watches sass files and updates mtime on main files for each change
       logToConsole: false // If true, will log to logger.error on errors
   }));
-  app.use('/css',express.static('../scss'));
+  app.use('/css',express.static('../static/scss'));
```

Comment:
I know we talked about this via email.  It doesn’t matter so much where SCSS files are located; however, they are _not_ static resources.  They have to be compiled.

Reply:
Good point, yes - I didn't consider that.  Guess there isn't really any existing place where the scss files would fit, so I'll revert it back to how it was originally.